src/components/select/Select.js

```diff
@@ -505,6 +505,10 @@ export default class SelectComponent extends Field {
   /* eslint-enable max-statements */
 
   get defaultValue() {
+    if (this.options.display === 'pdf' && (this.options.readOnly || this.root.formio?.submissionId)) {
+      this.component.defaultValue = '';
```

Comment:
This is an unusual change. What is the issue that you are fixing here, and why is it necessary to reset the defaultValue of the component to fix it?  Is there another way?